PocketKit/Sources/Sync/Crashlogger.swift

```diff
@@ -22,10 +22,25 @@ public struct Crashlogger {
     }
 
     public static func start(dsn: String) {
+        if isRunningTests() {
+            // We are in a test environment, lets not init sentry.
+            return
+        }
+
         SentrySDK.start { options in
             options.dsn = dsn
-            options.enableAutoPerformanceTracking = false
+            options.enableAutoSessionTracking = true
```

Comment:
What kinds of things does this track?

Reply:
@dskuza Release health https://docs.sentry.io/product/releases/health/